rosa_install_access_delete_clusters/rosa-shared-vpc-config.adoc

```diff
@@ -58,3 +58,17 @@ include::modules/rosa-sharing-vpc-creation-and-sharing.adoc[leveloffset=+1]
 include::modules/rosa-sharing-vpc-dns-and-roles.adoc[leveloffset=+1]
 include::modules/rosa-sharing-vpc-hosted-zones.adoc[leveloffset=+1]
 include::modules/rosa-sharing-vpc-cluster-creation.adoc[leveloffset=+1]
+
+[id="troubleshooting_shared-vpc_vpc-creation"]
+== Troubleshooting
+
+If your cluster fails to install, check these common troubleshooting issues:
+
+* Make sure you included a domain name.
+* Make sure the domain name does not include any spaces or capital letters.
```

Comment:
```suggestion
* Make sure your DHCP option set includes a domain name, and ensure that domain name does not include any spaces or capital letters.
```
You might also consider linking "DHCP option set" to https://docs.aws.amazon.com/vpc/latest/userguide/VPC_DHCP_Options.html